scalac-scoverage-plugin/src/main/scala/scoverage/plugin.scala

```diff
@@ -35,16 +34,23 @@ class ScoveragePlugin(val global: Global) extends Plugin {
         options.dataDir = opt.substring("dataDir:".length)
       } else if (opt.startsWith("extraAfterPhase:") || opt.startsWith("extraBeforePhase:")) {
         // skip here, these flags are processed elsewhere
-      } else {
+      } else if (opt.startsWith("useEnvironment:")) {
+        options.useEnvironment = opt.substring("useEnvironment:".length).toBoolean
```

Comment:
Can we have some error checking here? `.toBoolean` could potentially throw an error here.